### PR TITLE
Encapsulate most game state info

### LINKED ISSUE #2: Game state encapsulation
Encapsulate the game state, including player roles, game phases, and chat history.

### PR DESCRIPTION
**Pull request by Marko-Shevchuk**

### CODE REVIEW

File: Assets/Scripts/ChatMessage.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+[System.Serializable]
+public class ChatMessage
+{
+    private long senderId;
+    public string sender;
+    public MessageStyle.StyleType style;
+    public string message;
+    public float timeSent;
+}
+/*public class ChatManager : MonoBehaviour
```
Comment (ArsenOOO7) [inline Assets/Scripts/ChatMessage.cs]: why? :(

File: Assets/Scripts/PlayerProfile.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+using System;
+using UnityEngine;
+
+[System.Serializable]
+public class PlayerProfile
+{
+    public long id;
```
Comment (ArsenOOO7) [inline Assets/Scripts/PlayerProfile.cs]: ```public``` 😭

File: Assets/Scripts/enums/Alignment.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+using System;
+
+public class Alignment
+{
+    public enum AlignmentType { Civ, Maf }
```
Comment (ArsenOOO7) [inline Assets/Scripts/enums/Alignment.cs]: ```CIV```, ```MAF``` ? (uppercase)

File: Assets/Scripts/enums/GamePhase.cs [hunk 4]
```diff
@@ -0,0 +1,6 @@
+using System;
+
+public class GamePhase
+{
+    public enum PhaseType { NightChat, NightVote, DayChat, DayVote, GameOver }
```
Comment (ArsenOOO7) [inline Assets/Scripts/enums/GamePhase.cs]: Or is it according to some convention?

File: Assets/Scripts/enums/MessageStyle.cs [hunk 5]
```diff
@@ -0,0 +1,6 @@
+using System;
+
+public class MessageStyle
+{
+    public enum StyleType { Normal, Dead, Unknown, Warning, Error, Success, Day, Night, System1, System2, System3 } // I have no idea what I'm doing
```
Comment (ArsenOOO7) [inline Assets/Scripts/enums/MessageStyle.cs]: ```System2``` 😨

**Review by ArsenOOO7 — APPROVED** [review]
VERY GOOD JOB! (Maybe)

File: Assets/Scripts/enums/Alignment.cs (hunk 3 above)
Comment (ArsenOOO7) [reply Assets/Scripts/enums/Alignment.cs]: and wtf, why enum is nested into class

File: Assets/Scripts/enums/GamePhase.cs (hunk 4 above)
Comment (ArsenOOO7) [reply Assets/Scripts/enums/GamePhase.cs]: NIGHT_CHAT

File: Assets/Scripts/PlayerProfile.cs (hunk 2 above)
Comment (Marko-Shevchuk) [reply Assets/Scripts/PlayerProfile.cs]: My goals are beyond your understanding.

File: Assets/Scripts/enums/MessageStyle.cs (hunk 5 above)
Comment (Marko-Shevchuk) [reply Assets/Scripts/enums/MessageStyle.cs]: h1 h2 h3

File: Assets/Scripts/enums/GamePhase.cs (hunk 4 above)
Comment (Marko-Shevchuk) [reply Assets/Scripts/enums/GamePhase.cs]: Common sense was never an option.

**Review by JJerome-NM — APPROVED** [review]

**Pull request merged**